app/api/views/brief_responses.py

```diff
@@ -0,0 +1,113 @@
+from flask import jsonify
+from flask_login import current_user, login_required
+from app.api import api
+from app.api.helpers import abort, not_found
+from ...models import AuditEvent
+from app.api.services import brief_responses_service, audit_service, audit_types
+from app.api.helpers import role_required
+from ...datetime_utils import utcnow
+import rollbar
+
+
+@api.route('/brief-response/<int:brief_response_id>/withdraw', methods=['POST'])
```

Comment:
Should this be a DELETE? We are using DELETE's for soft-deletions, but they seem to be in the older app/main/views structure and not app/api/views.

Reply:
we could do - but then does the `/withdraw' part make sense

Reply:
looking further at the meaning of DELETE in the code, I believe this should be PUT.

My reasoning is that the DELETE keyword is used for a hard delete or setting a `status` column to 'deleted' in the app/main/views

open to suggestions

Reply:
My understanding of PUT (and PATCH) is the request body should contain the data the resource will now contain - so a PUT/PATCH may make more sense if the resource was a generic "update the brief response" resource.

And seeing as `egrep -Rn '(PUT|PATCH|DELETE)' app/ | grep 'api/views'` returns nothing though perhaps it's best left as POST for consistency sake.

Reply:
I suspect anytime we have a verb in a resource path that we're probably breaking from strict REST conventions anyway, but that sounds right to me. Although I'd make a slight adjustment and suggest `withdraw` is changed to `withdrawn_at`, i.e. from verb to the actual field name, as it would then use the data in the request (presumably a timestamp) to make the change, which seems more in tune with how REST views PUT, although it's splitting hairs a bit at this point.